MonoGame.Framework/Content/ContentManager.cs

```diff
@@ -476,11 +475,14 @@ public IServiceProvider ServiceProvider
 		}
 
         internal byte[] GetScratchBuffer(int size)
-        {            
+        {
             size = Math.Max(size, 1024 * 1024);
-            if (scratchBuffer == null || scratchBuffer.Length < size)
-                scratchBuffer = new byte[size];
-            return scratchBuffer;
+            return _scratchBufferPool.Get(size);
+        }
+
+        internal void ReturnScratchBuffer(byte[] buffer)
+        {
+            _scratchBufferPool.Return(buffer);
```

Comment:
How about just making `_scratchBufferPool` `internal` and let people call it directly?  Just to avoid these redirect methods.  Just make `ByteBufferPool` ctor take an optional minimum size.

Reply:
I am not sure if it's best to modify ByteBufferPool or use a second pool and let the two take different roads for contentManager & soundEffects. In every case I prefer having a layer that allows us to be more flexible with the internal implementation. Maybe having a IBufferPool with Get()/Return() or something like that.